package/rpm/magnum-plugins.spec

```
@@ -0,0 +1,119 @@
+Name: magnum-plugins
+Version: 2020.06.1513.g72597410
+Release: 1
+Summary: Plugins for the Magnum C++11/C++14 graphics engine
+License: MIT
+Source: %{name}-%{version}.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Requires: magnum, DevIL, libpng, libjpeg-turbo, freetype, assimp, faad2-libs, glslang, spirv-tools-libs, libwebp, libspng, openexr-libs
+BuildRequires: cmake, git, gcc-c++, DevIL-devel, libpng-devel, libjpeg-turbo-devel, freetype-devel, assimp-devel, faad2-devel, harfbuzz-devel, glslang-devel, spirv-tools-devel, libwebp-devel, libspng-devel, openexr-devel
+Source1: https://github.com/BinomialLLC/basis_universal/archive/refs/tags/1.15_final.zip
+Source2: https://github.com/zeux/meshoptimizer/archive/refs/tags/v0.18.zip
+
+%description
+Here are various plugins for the Magnum C++11/C++14 graphics engine -
+asset import and conversion, text rendering and more.
+
+%package devel
+Summary: MagnumPlugins development files
+Requires: %{name} = %{version}
+
+%description devel
+Headers and tools needed for the Magnum plugins collection.
+
+%prep
+%setup -c -n %{name}-%{version}
+
+%build
+unzip %{SOURCE1} -d %{_builddir}/
+unzip %{SOURCE2} -d %{_builddir}/
+
+mv %{_builddir}/meshoptimizer-0.18 %{name}-%{version}/src/external/meshoptimizer
+
+mkdir build && cd build
+# Configure CMake
+cmake ../%{name}-%{version} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=%{_prefix} \
+  -DBASIS_UNIVERSAL_DIR=%{_builddir}/basis_universal-1.15_final \
+  -DCMAKE_COLOR_DIAGNOSTICS=ON \
+  -DMAGNUM_WITH_ASSIMPIMPORTER=ON \
+  -DMAGNUM_WITH_ASTCIMPORTER=ON \
+  -DMAGNUM_WITH_BASISIMAGECONVERTER=ON \
+  -DMAGNUM_WITH_BASISIMPORTER=ON \
+  -DMAGNUM_WITH_BCDECIMAGECONVERTER=ON \
+  -DMAGNUM_WITH_CGLTFIMPORTER=ON \
+  -DMAGNUM_WITH_DDSIMPORTER=ON \
+  -DMAGNUM_WITH_DEVILIMAGEIMPORTER=ON \
+  -DMAGNUM_WITH_DRFLACAUDIOIMPORTER=ON \
+  -DMAGNUM_WITH_DRMP3AUDIOIMPORTER=ON \
+  -DMAGNUM_WITH_DRWAVAUDIOIMPORTER=ON \
+  -DMAGNUM_WITH_ETCDECIMAGECONVERTER=ON \
+  -DMAGNUM_WITH_FAAD2AUDIOIMPORTER=ON \
```

Given that `faad2` needs an extra RPM repository, I'd explicitly mark it as 

```suggestion
  -DMAGNUM_WITH_FAAD2AUDIOIMPORTER=OFF \
```

here and remove the dependencies, so the default installation is ~painless. I'll then include the setup for this dependency just in the docs.